erpnext/projects/doctype/project/project.py

```diff
@@ -1382,6 +1384,14 @@ def get_item_groups_subtree(self, item_group):
 
 		return self._item_group_subtree[item_group]
 
+	def validate_notification(self, notification_type=None, child_doctype=None, child_name=None, throw=False):
+		if notification_type == "Ready to Close":
+			if self.ready_to_close and self.status == "To Close":
+				return True
+			else:
+				frappe.throw(_("Cannot send {0} notification because Repair Order status is not 'To Close'")
+					 .format(notification_type))
```

Comment:
write condition when notification should not be sent... throw only if argument asks for it. return true if no condition catches any problem